frontend/src/scenes/feature-flags/activityDescriptions.tsx

```diff
@@ -131,6 +136,24 @@ const featureFlagActionsMapping: Record<
         }
 
         if (isMultivariateFlag) {
+            Object.keys(filtersAfter.payloads).forEach((key: string) => {
+                const changedPayload = filtersAfter.payloads[key]?.toString() || null
```

Comment:
not sure because js is weird: Does this work as expected for both string and bool type keys?

Reply:
are you asking if something like this works?

<img width="188" alt="Screen Shot 2023-03-20 at 11 40 20 AM" src="https://user-images.githubusercontent.com/25164963/226391703-c4c6a363-e9de-4882-a71a-9b5108851b88.png">

Reply:
Everything works, I meant this: 

<img width="335" alt="image" src="https://user-images.githubusercontent.com/7115141/226400305-932720f5-cd86-45b7-88fd-f07b5782232d.png">


And I really don't know what to say now, all of it works 😂

Reply:
yeah you can also see it's typed as a `string` here `forEach((key: string)`